integrations/airflow/README.md

```diff
@@ -42,40 +42,27 @@ $ python3 setup.py install
 
 ## Configuration
 
-The library depends on a _backend_. A `Backend` is configurable and lets the library know where to write dataset, job, and run metadata.
 
-### Backends
-
-* `HTTP`: Write metadata to Marquez
-* `FILE`: Write metadata to a file (as `json`) under `/tmp/marquez`
-* `LOG`: Simply just logs the metadata to the console
-
-By default, the `HTTP` backend will be used (see next sections on configuration). To override the default backend and write metadata to a file, use `MARQUEZ_BACKEND`:
+### `HTTP` Backend Environment Variables
 
-```
-MARQUEZ_BACKEND=FILE
-```
+`marquez-airflow` uses OpenLineage client to push data to Marquez.
 
-> **Note:** Metadata will be written to `/tmp/marquez/client.requests.log`, but the location can be overridden with `MARQUEZ_FILE`.
+OpenLineage client depends on environment variables:
 
-### `HTTP` Backend Authentication
+* `OPENLINEAGE_URL` - point to service which will consume OpenLineage events
+* `OPENLINEAGE_PRODUCER` - name of producer that client will send along with your events
+* `OPENLINEAGE_API_KEY` - set if consumer of OpenLineage events requires `Bearer` authentication key
 
-The `HTTP` backend supports using API keys to authenticate requests via `Bearer` auth. To include a key when making an API request, use `MARQUEZ_API_KEY`:
+For backwards compatibility, `marquez-airflow` also support configuring
+target URL via `MARQUEZ_URL` env variable.
 
 ```
-MARQUEZ_BACKEND=HTTP
-MARQUEZ_API_KEY=[YOUR_API_KEY]
```

Comment:
Is `MARQUEZ_API_KEY` backward compatible?

Reply:
Yes. Although, clarified this and moved the change to source naming PR, as the change was actually made there. 
https://github.com/MarquezProject/marquez/pull/1453/files#diff-e029af63887ebdd96a81800f30b5829f330eb6e659a17d623135a429b97f7a93